packages/shadow-dom/src/findInsertionPoint.test.ts

```diff
@@ -65,6 +85,26 @@ describe('findInsertionPoint', () => {
       expect(findInsertionPoint(renderer, styleSheet)).toBe(null);
     });
 
+    it('handles insertionPoint in otherwise empty array', () => {
+      const insertionPoint = new CSSStyleSheet() as ExtendedCSSStyleSheet;
+
+      const renderer = createRendererMock([insertionPoint]);
+      const styleSheet = createStyleSheetMock('m', { m: '(max-width: 3px)' });
+
+      expect(findInsertionPoint(renderer, styleSheet, insertionPoint)).toBe(insertionPoint);
+    });
+
+    it('handles insertionPoint with other (non-Griffel) stylesheets', () => {
+      const insertionPoint = new CSSStyleSheet() as ExtendedCSSStyleSheet;
+      const other1 = new CSSStyleSheet() as ExtendedCSSStyleSheet;
+      const other2 = new CSSStyleSheet() as ExtendedCSSStyleSheet;
+
+      const renderer = createRendererMock([other1, insertionPoint, other2]);
```

Comment:
This is actually incorrect - since the `renderer` adoptedStylesheets would actually never contain non-Griffel created stylesheets. The other stylesheets would only be present in the shadow root's adoptedStylesheets collection.

Reply:
Removed